autoware.repos

```diff
@@ -76,6 +76,10 @@ repositories:
     type: git
     url: https://github.com/autowarefoundation/sample_sensor_kit_launch.git
     version: main
+  sensor_kit/external/awsim_sensor_kit_launch: # TODO: Integrate into sample_sensor_kit_launch
+    type: git
+    url: https://github.com/RobotecAI/awsim_sensor_kit_launch.git
```

Comment:
Is it possible transfer the repository to [autowarefoundation](https://github.com/autowarefoundation) directly put it to `sensor_kit` folder?

Reply:
It might be possible, but as written in the comment, he says this will be integrated into `sample_sensor_kit_launch` (or `awf_reference_sensor_kit_launch`?) and removed in the future.
@xmfcx What makes you want to transfer it to AWF now?

Reply:
I've missed the comment
> Note that this will be integrated to sample_sensor_kit_launch in the future.

from the description. It's ok for now, thanks.